docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+* **Allowed file extensions** - Extensions of files separated by commas. You can only upload files with the specified file extensions.
+
 Campaign settings
 *****************
 
+.. image:: images/campaign-settings.png
+  :width: 600
+  :alt: Screenshot showing Campaign Settings Configuration in Mautic
+
+* **Wait time before retrying a failed action** - If for any reason a Campaign action doesn't execute, this is the length of time Mautic waits before trying again.
+
+* **Use date range for all views** - When viewing a Campaign, the actions, conditions, decisions, and Contacts tabs is based on the date range provided.
+
+* **Use summary statistics** - Improves performance when viewing a Campaign with thousands of events per day by using summarized data.
+
 Email settings
 **************
 
 Mail send settings
 ==================
 
+.. image:: images/mail-send-settings.png
+  :width: 600
+  :alt: Screenshot showing Mail Send Settings Configuration in Mautic
+
+* **Name to send mail as** - The default name Emails come from. This is typically something like ``{YourCompany}`` Marketing Team or ``{YourCompany}``.
+
+* **Email address to send mail from** - The Email address for the name you’re sending mail from. The address displays in the From: field when your Contacts receive your Emails.
+
+.. note::
+
+   Ensure that you configure your sender domain, DKIM, bounce, and click tracking domains. For more information, see Email domains document.
+
+* **Reply to address** -  To have Contacts reply to a different address than the address defined in Email address to send mail from, add the desired address to receive replies. This is the default address where messages are sent whenever Contacts reply to messages sent from Mautic. If this field is blank, the address specified in **Email address to send mail** from is used. This is useful if your configured sender domain, which you use in the from address, contains a subdomain that doesn't have MX records or is otherwise an address that can't receive Emails.
+
+* **Custom return path (bounce) address** - Set a custom return path/bounce Email for Emails sent from the system. Note that some mail transports, such as GMail, won'the support this.
+
+* **Mailer is owner** - If Contacts in Mautic have ``Owners``, select Yes to display the owner as sending all Emails to the Contacts they’re listed as the owner for. 
+
+.. note:: 
+
+    Mailer is owner overrides any other name or Email to send mail from, including the default and individual Emails. Every Contact owner’s domain must have SPF and DKIM records. You can see this configuration for individual Emails, rather than globally.
+    For more information see :doc:`Mailer is owner</channels/emails>`
+
+* **Service to send mail through** - For most Mautic customers, this should remain as ``Mautic``. Your Emails route through the Mautic's Sparkpost account. If you are sending through your own Email service, select the service and enter your credentials. 
```

Comment:
```suggestion
* **Service to send mail through** - Select the Email service provider you use, and enter your credentials. 
```

This first part was Acquia-specific - please watch out for this if you are pulling in documentation from SaaS providers.